packages/lisk-transactions/src/transactions/base.ts

```diff
@@ -358,32 +357,39 @@ export abstract class BaseTransaction {
 		}
 		// Verify multisignatures
 		if (
-			Array.isArray(sender.multisignatures) &&
-			sender.multisignatures.length > 0 &&
-			sender.multimin
+			!(
+				Array.isArray(sender.multisignatures) &&
+				sender.multisignatures.length > 0 &&
```

Comment:
`sender.multisignatures.length > 0` is true, we can assume it's an array?

Reply:
gives `possibly undefined` without any check. I can check if `sender.multisignatures` exists instead.